pyproject.toml

```diff
@@ -75,6 +75,7 @@ tests = [
     "black",
     "flake8",
     "factory-boy",
+    "rosetta",
```

Comment:
I think the library is called `django-rosetta` on pypi (https://pypi.org/project/django-rosetta/). Locally the tests don't work for me: `ModuleNotFoundError: No module named 'rosetta'